### DISCUSSION TITLE
[Query] Do we have any place where we can ask for referrals ?

### DISCUSSION THREAD

**Discussion by OldSlayer1903** [Green-Software-Foundation/discussions#170] [Ask Anything]
Hey People, 
It's been more than 2 years of me following the Green-Software-Foundation initiatives and I've seen the good amount of growth, in terms of projects and the organizations coming together to work around a better more greener future. 

* I was wondering while we have created this great community and we have ample amount of great technologist, do we think about understanding if it could help us change roles or grow into a role that aligns with our aspirations.

* Looking at market trends and layoffs ongoing in a lot of orgs, would it help if we can open a space or have a small thread to search and help fellow affected folks.

**Comment by NAMRATA-WOKE** [comment]
@OldSlayer1903 would love for you to elaborate on questions you might have vis-a-vis changing roles or shaping current roles to make more room for green software projects. Are you also looking for easier ways to identify career opportunities in green software?
